tests/examples/aws-java-webserver/app/src/main/java/webserver/App.java

```diff
@@ -23,7 +23,7 @@ public static void main(String[] args) {
 
     public static void stack(Context ctx) {
         final var amiResult = Ec2Functions.getAmi(GetAmiArgs.builder()
-                .filters(GetAmiFilter.builder()
+                .filters(GetAmiFilterArgs.builder()
```

Comment:
Any idea why is this changing? This feels like a breaking change that was not called for.

Reply:
Figure out - this is because GetAmiArgs is now InputShape, so all the nested types get named Args. Perfect.